packages/log/src/LogExport.ts

```diff
@@ -14,11 +14,19 @@ export const DEFAULT_PATH_BLACKLIST: string[][] = [
   ['dashboardData', '*', 'openedMap'],
   ['draftManager', 'draftStorage'],
   ['layoutStorage'],
+  ['storage'],
+
+  // Below are confirmed enterprise specific keys, and will be moved in DH-20410
```

Comment:
`draftManager` is also an enterprise specific key.
Okay to have this here for now, but on Enterprise side we should just add to the list at https://github.com/deephaven-ent/iris/blob/rc/sanluis/web/client-ui/src/utils/exportEnterpriseLogs.ts#L18